frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -257,89 +257,76 @@ describe("scenarios > dashboard > dashboard drill", () => {
       .within(() => cy.findByText("foo"));
   });
 
-  it("should pass multiple filters for numeric column on drill-through (metabase#13062)", () => {
-    // Preparation for the test: "Arrange and Act phase" - see repro steps in #13062
-    // 1. set "Rating" Field type to: "Category"
+  ["field value", "card title"].forEach(testCase => {
```

Comment:
Because of the `if`s inside this test, customizing quite a bit for each `testCase`, it starts looking more like this should be a 

`describe('should pass multiple filters for numeric column on drill-through', () => {
   it('when ??? is field value`, () => { ... }');
   
      it('when ??? is card title`, () => { ... }');
}`

??? = not clear to me what to call.

There will be repeated code for each `it`, sure, but we can refactor that using this other, just more typical, structure for tests. 

See below.

Reply:
Done.
Thanks for the suggestion.

![image](https://user-images.githubusercontent.com/31325167/120230868-7faabc00-c250-11eb-9640-525e1a158731.png)